src/index.ts

```diff
@@ -228,3 +241,9 @@ export async function init(config: IClientConfig): Promise<IEppoClient> {
 export function getInstance(): IEppoClient {
   return EppoJSClient.instance;
 }
+
+export async function getFlagConfigurations(
```

Comment:
The idea is that this function is called somewhere server side to get the flag config json, and then the JSON is passed in to the init function in `config.flagConfigurations`